packages/sources/uscpi-one/src/endpoint/series.ts

```diff
@@ -0,0 +1,77 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+
+export const supportedEndpoints = ['series']
+
+export const endpointResultPaths = {
+  example: 'series',
```

Comment:
Here, the key should be the endpoint name (example -> series) and the value should be the path.

Reply:
done